tests/DynamoDbGlobalScopeTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace BaoPham\DynamoDb\Tests;
+
+use Carbon\Carbon;
+use BaoPham\DynamoDb\DynamoDbQueryBuilder;
+
+/**
+ * Class DynamoDbTimestampTest
```

Comment:
Correct - seems to work with or without it in the class, as it's inherited.  This has been updated.  Should be ready to merge, I believe I updated all of the items you requested.